app/components/Button/Button.module.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../styles/vars.scss";
+
+.button {
+  background: var(--button-background);
```

Comment:
Thinking we could use the same css vars as the dark mode toggle for this button actually. Might just be worth renaming it to something more generic?

But if you wanted to keep the seperate css vars, can you please follow the same bg/hover colours as the toggle? I know it's different from the design, sorry.

This one -> `--dark-mode-toggle`

And same for the hover -> `--dark-mode-toggle-hover`